src/models/contributors.d.ts

```diff
@@ -0,0 +1,29 @@
+export interface IContributorResponse {
+  login: string;
+  id: number;
+  node_id: string;
+  avatar_url: string;
+  gravatar_id: string | null;
+  url: string;
+  html_url: string;
+  followers_url: string;
+  following_url: string;
+  gists_url: string;
+  starred_url: string;
+  subscriptions_url: string;
+  organizations_url: string;
+  repos_url: string;
+  events_url: string;
+  received_events_url: string;
+  type: string;
+  site_admin: boolean;
+  contributions: number;
+  email?: string;
```

Comment:
I would suggest making this `email: string | null` to stay consistent with the rest of your project